AstroSaveScenario.py

```diff
@@ -0,0 +1,204 @@
+    Logger.logPrint("\t2) New folder in a custom path")
+
+    choice = input()
+    while choice not in ('1', '2'):
+        Logger.logPrint(f'\nPlease choose 1 or 2')
+        choice = input()
+        Logger.logPrint(f'copy_choice {choice}', 'debug')
+
+    if choice == '1':
+        # Winpath is needed here because Windows user can have a custom Desktop location
+        save_path = utils.get_windows_desktop_path()
+    elif choice == '2':
+        Logger.logPrint(f'\nEnter your custom folder path:')
+        save_path = input()
+        Logger.logPrint(f'save_path {save_path}', 'debug')
+
+    return utils.join_paths(save_path, utils.create_folder_name('AstroSaveFolder'))
+
+
+def ask_custom_folder_path() -> str:
+    Logger.logPrint(f'\nEnter your custom folder path:')
+    path = input()
+    Logger.logPrint(f'save_folder_path {path}', 'debug')
+
+    if utils.is_folder_a_dir(path):
+        return path
+    else:
+        Logger.logPrint(f'\nWrong path for save folder, please enter a valid path : ')
+        return ask_custom_folder_path()
+
+
+def print_save_from_container(save_list):
+    """ Displays the human readable saves of a container """
+    for i, save in enumerate(save_list):
+        Logger.logPrint(f'\t {str(i+1)}) {save.save_name}')
+
+
+def ask_saves_to_export(save_list):
+    Logger.logPrint('Extracted save list :')
+    print_save_from_container(save_list)
+    Logger.logPrint('\nWhich saves would you like to convert ? (Choose 0 for all of them)')
+    Logger.logPrint('(Multi-convert is supported. Ex: "1,2,4")')
+
+    maximum_save_number = len(save_list)
+    saves_to_export = ask_for_multiple_choices(maximum_save_number)
+
+    return saves_to_export
+
+
+def ask_for_multiple_choices(maximum_value) -> list:
+    """ Let the user choose multiple numbers between 0 and a maximum value
+
+    If the user choice is 0 then return an array with all values,
+    the user choices are reduce by 1 in order to match future array indexes
+
+    :Example:
+
+    User choices:
+    -  [1,2,4]  - returns 0, 1, 2
+    -  [0]      - returns all choices
+
+    :return: The list of numbers
+    :exception: None (repeat until the choices are valid)
+    """
+    choices = []
+    while not choices:
+        choices = input()
+        choices = process_multiple_choices_input(choices, maximum_value)
+        try:
+            verify_choices_input(choices)
+        except ValueError:
+            choices = []
+            Logger.logPrint(f'Please use only values between 1 and {maximum_value} or 0 alone')
+
+        if choices == [-1]:
+            return list(range(0, maximum_value))
+        else:
+            return choices
+
+
+def process_multiple_choices_input(choices, max_value) -> list:
+    choices = choices.split(',') if utils.rcontains(',', choices) else choices
+    choices = map(lambda x: int(x), choices)
+    choices = [number - 1 for number in choices if number >= 0 or number < max_value]
```

Comment:
Compared to original code, a false value inside a set of correct value does not raise an error anymore.
Please do it